src/main.rs

```diff
@@ -167,29 +176,36 @@ fn main() {
     if !template.is_empty() {
         url.push_str(&format!("&template={}", template));
     }
-    println!("Click on this link to generate your PR {}", url);
+
+    // Check if -o flag is specified and open the URL in the default browser
+    let open_in_browser = matches.is_present("open");
+
+    if open_in_browser {
+        if webbrowser::open(&url).is_ok() {
+            println!("Opened the URL in the default browser: {}", url);
+        } else {
+            eprintln!("Failed to open the URL in the browser. Here is the link: {}", url);
+        }
+    } else {
+        println!("Click on this link to generate your PR: {}", url);
+    }
 }
 
 fn get_config_value<'a>(matches: &clap::ArgMatches, config: Option<&RcFile>, key: &str) -> String {
-    // Check matches first
     let value_from_matches: Option<&str> =
         matches.get_one::<String>(key).map(|s: &String| s.as_str());
 
-    // If no value found, check the config
     let value_from_config: Option<&str> = config.and_then(|cfg: &RcFile| {
         cfg.defaults
             .as_ref()
             .and_then(|d: &Defaults| d.values.get(key))
             .map(|s: &String| s.as_str())
     });
 
-    // Combine both options
     value_from_matches
```

Comment:
Don't remove comments that were already present

Reply:
Added the comments back